src/current/v23.2/migration-strategy-live-migration.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Migration Strategy: Live Migration"
+summary: Learn about the 'Live Migration' data migration strategy
+toc: true
+docs_area: migrate
+---
+
+This topic discusses the *live migration* strategy for [migrating your data to CockroachDB]({% link {{ page.version.version }}/migration-overview.md %}). See [Develop a migration plan]({% link {{ page.version.version }}/migration-overview.md %}#develop-a-migration-plan) for detailed information on deciding which migration strategy will work best for your application.
+
+During a live migration you maintain two production databases (the *source* and *target* database) for a period of time, and replicate data between them until a final cutover to the target database. The purpose of a live migration is to minimize downtime during this final cutover period and therefore, maintain service continuity.
+
+A live migration consists of three distinct phases:
+
+- The initial phase where the source database is the *source of truth*: the database used by the application to read and write data, and the source for replicating the data to the target database.
+- The cutover phase where CockroachDB is the source of truth and the source database is the secondary database.
+- The final phase where the source database is removed as a replication target and failover database.
+
+There are many possible approaches to performing a live migration. In this topic, we describe two example approaches that have been successful with CockroachDB: consistent cutover, and immediate cutover.
+
+The consistent cutover approach uses only one database that is writable and consistently readable at the application level, with the data replicated on the back end by another service. When using the consistent cutover approach, one database is the source of truth at any point in time. During the cutover phase, any current application transactions that use the source database are allowed to complete, and the replication service is allowed to catch up. Then new application sessions are allowed to start using the target database, and the replication service replicates data from the target database to the source database. If any problems arise during the cutover phase and you have to revert back to the initial phase, the same process repeats, moving from the target database back to the source database.
+
+With the immediate cutover approach, the application performs dual writes to the source and target database during both the initial and cutover phases. This approach can lead to incorrect or inconsistent data between the two databases if there are issues when reading or writing data to one or both of the underlying databases, and may require work after cutover to correct any unacceptable data inconsistencies.
```

Comment:
Raising this because the terms "consistent cutover" and "immediate cutover" appear to be derived from the LMS, and should ideally be consistent with what the LMS allows. It seems that with the LMS, it's possible to perform a consistent cutover while using a shadowing method (strict sync) that dual writes to both databases. Conversely, it's also possible to perform an immediate cutover even without dual writing. 

I wonder if we might avoid opening a can of worms by simply differentiating "consistent" and "immediate" cutover in terms of whether you wait for replication to catch up on the target (consistent) or don't (immediate), and not talk about whether there is dual writing. @cotedm @ianjevans thoughts?

Reply:
i +1 this comment. you can do a "consistent cutover" with a "shadowing" approach if you implement an "atomic pause" and validation check (see above). 

really, "consistent cutover" is an atomic pause. there's a number of strategies involved in getting data correct on both sides.

Reply:
Yeah this is a good point. Basically generally describe what a consistent cutover and immediate cutover is in this doc and then in the LMS docs we provide our recommendation on how to do it with our tools. @ianjevans let me know if you need help with getting this to be a bit more generic as @taroface described.